src/main/typescript/node_modules/@atomist/rug/tree/TreeHelper.ts

```diff
@@ -66,3 +66,15 @@ export class TreeHelper {
         return r
     }
 }
+
+export abstract class AddressedNodeSupport implements Addressed {
+
+    private _address: string = null
```

Comment:
This code ended up changed quite a bit anyway, as I found and fixed a bug in address handling